### ISSUE TITLE
New instruction naming consistency:

### ISSUE THREAD

**Issue by at67** [at67/ROMvX0#21]
An issue that I am not so happy with is that some of the instructions, (mostly mine, if not all mine), are named in a not so consistent manner and some of their operands are fetched in a not so consistent fashion.

This will need to be addressed before release and probably should be done as a final pass just before release, (as it will probably require non trivial changes to compilers and therefore we should do it only once if possible).

Low priority for now.
